packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    :data-testid="`collaborator-item-${share.collaborator.name}`"
+    :data-testid="`collaborator-${isUser ? 'user' : 'group'}-item-${share.collaborator.name}`"
```

Comment:
`guest` and `remote` could be possibilities, too? (Imagine inviting both the admin in your and a federated ownCloud instance)

Reply:
fact, as discussed we should handle that too, for now we keep it as it is and take care in a separate pr.